hr_contract_hourly_rate/models/hr_contract.py

```diff
@@ -19,35 +18,24 @@
 #
 ##############################################################################
 
-from openerp.osv import fields, orm
+from openerp import models, fields, api, exceptions, _
 
 
-class hr_contract(orm.Model):
+class hr_contract(models.Model):
     _inherit = 'hr.contract'
-    _columns = {
-        'salary_computation_method': fields.selection(
-            [
-                ('wage', 'Annual Wage'),
-                ('hourly_rate', 'Hourly Rate')
-            ],
-            'Salary Computation Method',
-            help="Whether to use the annual wage or an hourly rate "
-            "for computation of payslip.",
-            required=True,
-        ),
-    }
-    _defaults = {
-        'salary_computation_method': 'wage',
-    }
 
-    def get_job_hourly_rate(
-        self, cr, uid,
-        date_from, date_to,
-        contract_id,
-        job_id=False,
-        main_job=False,
-        context=None,
-    ):
+    salary_computation_method = fields.Selection(
+        [('yearly', 'Annual Wage'),
+         ('monthly', 'Annual Wage'),
+         ('hourly', 'Hourly Wage')],
+        string='Salary Computation Method',
```

Comment:
Yes, but the conditions below, like `if contract.salary_computation_method != 'hourly':` , will fail on V7 items for every case. This means that without properly migrating `wage` to `yearly` or `hourly_rate` to `hourly`, the behaviour will be different before and after migrating for every case.

Is it possible to, at least, add a comment in the README in order to describe this change in the selection? That would be enough for me to know it's documented somewhere.